### DISCUSSION TITLE
Is there any limit on number of coder_agent on docker template?

### DISCUSSION THREAD

**Discussion by jbalzar** [coder/coder#10665] [General]
I'm trying to create a template with multiple agents and docker containers. For some reason, I can only have 4 running coder_agent. When I try to create the 5th coder_agent, the ports are just disabled with a loading icon and I can't access the terminal.

**Comment by matifali** [comment]
I can not reproduce this. I can run a workplace with 6 coder agents and no issues. Coder version 2.3.3.
![image](https://github.com/coder/coder/assets/10648092/40de0453-b860-4611-b873-b673fd625a53)
